games/datingsim/master.py

```diff
@@ -818,10 +880,11 @@ def compute_scores(self, episode_interactions: Dict) -> None:
             self.log_episode_score(BENCH_SCORE, np.nan)  # metric not applicable
         else:
             self.log_episode_score(METRIC_ABORTED, 0)
-            if total_agreements > 0:
+            if success > 0: # basically it is 1 or 0
```

Comment:
> Just an idea, could we make success True/False? Or would it mess up the code a lot? Again, cosmetic change but I'm wondering ^^

It's better to leave it 0/1 because we use it in calculating the bench score, more convenient